pages/resources.js

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types';
+import Content from 'components/Content/Content';
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+
+import ResourceCard from 'components/Cards/ResourceCard/ResourceCard';
+import Pagination from 'components/Pagination/Pagination';
+import { getResourcesPromise } from 'common/constants/api';
+import styles from './resources.css';
+
+const ResourcesContent = ({ resources, currentPage, totalPages }) => {
+  // For local development, only do server-side data fetching because of CSRF policy
+  const resourceMarkup = resources.map(resource => (
+    <ResourceCard
+      key={resource.id}
+      description={resource.notes}
+      downvotes={resource.downvotes}
+      upvotes={resource.upvotes}
+      href={resource.url}
+      name={resource.name}
+    />
+  ));
+
+  return (
+    <div>
```

Comment:
In terms of removing extra mark-up, we can make this a `section` and just pass `ResourcesContent` as the only item in the columns array of `Content`.

Also, you could remove the `div` wrapping `Pagination`